doc/Pacemaker_Explained/en-US/Ch-ACLs.txt

```diff
@@ -0,0 +1,334 @@
+ indexterm:[access control list,acl_permission,reference]
+
+|xpath
+|An https://www.w3.org/TR/xpath-10/[XPath] specification selecting an XML
+ element in the CIB to which the permission applies. Attributes may be
+ specified in the XPath to select particular elements, but the permissions
+ apply to the entire element.
+ (Exactly one of +object-type+, +xpath+, and +reference+ must be specified for
+ a permission.)
+ indexterm:[xpath,acl_permission]
+ indexterm:[access control list,acl_permission,xpath]
+
+|====
+
+[IMPORTANT]
+====
+* Permissions are applied to the selected XML element's entire XML subtree
+  (all elements enclosed within it).
+
+* Write permission grants the ability to create, modify, or remove the element
+  and its subtree, and also the ability to create any "scaffolding" elements
+  (enclosing elements that do not have attributes other than an ID).
+
+* Permissions for more specific matches (more deeply nested elements) take
+  precedence over more general ones.
+
+* If multiple permissions are configured for the same match (for example, in
+  different roles applied to the same user), any +deny+ permission takes
+  precedence, then +write+, then lastly +read+.
+====
+
+== ACL Targets and Groups ==
+
+ACL targets correspond to user accounts on the system.
+
+.Properties of an ACL Target
+[width="95%",cols="1m,<3",options="header",align="center"]
+|====
+
+|Attribute
+|Description
+
+|id
+|The name of a user on the system (required)
+ indexterm:[id,acl_target]
+ indexterm:[access control list,acl_target,id]
+
+|====
+
+ACL groups may be specified, but are not currently used by Pacemaker. This is
+expected to change in a future version.
+
+.Properties of an ACL Group
+[width="95%",cols="1m,<3",options="header",align="center"]
+|====
+
+|Attribute
+|Description
+
+|id
+|The name of a group on the system (required)
+ indexterm:[id,acl_group]
+ indexterm:[access control list,acl_group,id]
+
+|====
+
+Each +acl_target+ and +acl_group+ element may contain any number of +role+
+elements.
+
+.Properties of an ACL Role Reference
+[width="95%",cols="1m,<3",options="header",align="center"]
+|====
+
+|Attribute
+|Description
+
+|id
+|The +id+ of an +acl_role+ element that specifies permissions granted to the
+ enclosing target or group
+ indexterm:[id,role]
+ indexterm:[access control list,role,id]
+
+|====
+
+[IMPORTANT]
+====
+The +root+ and +hacluster+ user accounts always have full access to the CIB,
+regardless of ACLs. For other user accounts, when +enable-acl+ is true,
+permission to all parts of the CIB is denied by default (permissions must be
+explicitly granted).
+====
+
+== ACL Examples ==
+
+[source,XML]
+----
+<acls>
+
+   <acl_role id="read_all">
+       <acl_permission id="read_all-cib" kind="read" xpath="/cib" />
+   </acl_role>
+
+   <acl_role id="operator">
+
+       <acl_permission id="operator-maintenance" kind="write"
+           xpath="//crm_config//nvpair[@name='maintenance-mode']" />
```

Comment:
It's not necessary since this is just an example , but the demonstration would be more complete to also have:
"//nvpair[@name='maintenance']"

, which probably is the preferred way for "operator"s/"administrator"s to do node/resource maintenance :-)